.github/workflows/dbt_slim_ci.yml

```diff
@@ -68,6 +68,22 @@ jobs:
       - name: dbt test initial model(s)
         run: "dbt test $PROFILE --select state:new state:modified --exclude tag:prod_exclude tag:remove --defer --state . --project-dir ${PROJECT_DIR}"
 
+      - name: Check for new blockchain additions to dex_trades
```

Comment:
These steps and below shouldn't be needed since a change to the dex_trades model will retrigger a run and then a test anyway or am I missing something?

Reply:
we don't join to prices until `dex_trades` spell via the enhance macro & when we add a new chain, it gets added to `dex_base_trades` prior to the enhance step. due to this, we won't know if we're missing prices since we don't have the data yet. CI steps will only runs what's edited, so it stops at `dex_base_trades`. we'd have to force a change to `dex_trades` to see that run each time, not really scalable.

adding a few steps to generic CI workflow isn't ideal for dex specifically. the other idea was introducing a new workflow and only run on PRs that hit this path indicated below.

i guess we could have a dex-specific CI workflow, where we essentially use this updated workflow fully, then keep the old one as-is?